app/src/androidTest/java/com/github/houseorganizer/houseorganizer/FirebaseTestsHelper.java

```diff
@@ -135,6 +138,28 @@ protected static void createTestTaskList() throws ExecutionException, Interrupte
         FirestoreTask.storeTaskList(taskList, db.collection("task lists"), "task_list_1");
     }
 
+    /**
+     * This method will create the three households
+     */
+    protected static void createHouseholds() throws ExecutionException, InterruptedException {
+        createTestHouseholdOnFirestoreWithName(TEST_HOUSEHOLD_NAMES[0], TEST_USERS_EMAILS[0],
+                Arrays.asList(TEST_USERS_EMAILS[0], TEST_USERS_EMAILS[1]), TEST_HOUSEHOLD_NAMES[0]);
+
+        createTestHouseholdOnFirestoreWithName(TEST_HOUSEHOLD_NAMES[1], TEST_USERS_EMAILS[0],
+                Arrays.asList(TEST_USERS_EMAILS[0], TEST_USERS_EMAILS[2]), TEST_HOUSEHOLD_NAMES[1]);
+
+        createTestHouseholdOnFirestoreWithName(TEST_HOUSEHOLD_NAMES[2], TEST_USERS_EMAILS[1],
+                Arrays.asList(TEST_USERS_EMAILS[1], TEST_USERS_EMAILS[2], TEST_USERS_EMAILS[3],
+                        TEST_USERS_EMAILS[4], TEST_USERS_EMAILS[5], TEST_USERS_EMAILS[6]),
+                TEST_HOUSEHOLD_NAMES[2]);
+    }
+
```

Comment:
Maybe it would be great to have an external way to know which user are in each house other than having to go inside the helper class
This could be done later though